modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/UserAccountResourceTest.java

```diff
@@ -2281,6 +2288,33 @@ private void _testPostUserAccountWithImageExternalReferenceCode()
 		Assert.assertTrue(postUserAccount.getImageId() > 0);
 	}
 
+	private void _testPostUserAccountWithObjectValidation() throws Exception {
+		ObjectDefinition objectDefinition =
+			_objectDefinitionLocalService.
+				getObjectDefinitionByExternalReferenceCode(
+					"L_USER", testCompany.getCompanyId());
+
+		ObjectValidationRule objectValidationRule =
+			_objectValidationRuleLocalService.addObjectValidationRule(
+				StringPool.BLANK, _testUser.getUserId(),
+				objectDefinition.getObjectDefinitionId(), true,
+				ObjectValidationRuleConstants.ENGINE_TYPE_DDM,
+				LocalizedMapUtil.getLocalizedMap("This name is invalid."),
+				LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
+				ObjectValidationRuleConstants.OUTPUT_TYPE_FULL_VALIDATION,
+				"NOT(isEmpty(alternateName))", true, Collections.emptyList());
+
+		UserAccount postUserAccount = userAccountResource.postUserAccount(
+			_randomUserAccount(
+				userAccount -> userAccount.setUserAccountContactInformation(
```

Comment:
here is not enough 
`userAccount -> userAccount.setUserAccountContactInformation(null)`

instead of
`userAccount -> userAccount.setUserAccountContactInformation(() -> null)`
?

Reply:
I also checked in intellij, it will not compile with your change.